backend/src/facilities/fastify.js

```diff
@@ -31,6 +31,8 @@ const fastifyInstance = (options) => {
   app
     .decorate("verifyJWT", (req, res, done) => {
       const token = getBearerTokenFromRequest(req);
+      req.log.debug(token);
```

Comment:
if this is staying in, a message to say what is being dumped would be nice.
`the token from x is:`

Reply:
Not necessary, I'll removed it